src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -82,14 +82,14 @@ const LoginForm: React.FunctionComponent<PropsType> = ({
   };
   return (
     <div className="form-container">
-      <h2>Log in</h2>
       <div className="secondary-action">
         {`Don't have an account yet?`}
         <br />
         <span className="link" onClick={displayRegisterForm}>
           Register instead!
         </span>
       </div>
```

Comment:
Can we move this below the login form as well, and potentially put it in a `<small></small>` (or equivalent CSS)